kinetic/distribution.yaml

```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
+# ROS distribution file
+# see REP 143: http://ros.org/reps/rep-0143.html
+---
+release_platforms:
+  fedora:
+  - '22'
+  - '23'
```

Comment:
Right, I don't see any way around it. That's why I mentioned implementing some logic in Bloom and/or the buildfarm to use the "closest" version to the target platform version. I would say that 99% of the source RPMs built for ROS packages right now are the same across f20/f21/f22, so we could save generation time by skipping it all together if the rosdep keys resolve to the same RPM package names. If the version search logic was implemented in the buildfarm, this would also allow us to spin up additional platform versions after release - only the 1% of packages with differing keys would need to re-release in bloom - a very small price to pay, all things considered. If they aren't in the core package set, they could simply be left out until the package rev'd on its own.

I don't really want to get into this now, though. I'm sure we can poke holes in the idea in another thread.

For now, let's conclude that f23 and f24 should be listed here for ROS-K. Sound good?